src/cmd/sql_cmd_test.cc

```diff
@@ -270,6 +270,61 @@ TEST_F(SqlCmdTest, load_data) {
     unlink(file_name.c_str());
 }
 
+#if defined(__linux__)
+TEST_P(DBSDKTest, CreateFunction) {
+    auto cli = GetParam();
+    cs = cli->cs;
+    sr = cli->sr;
+    hybridse::sdk::Status status;
+    if (cs->IsClusterMode()) {
+        sr->ExecuteSQL("SET @@execute_mode='online';", &status);
+        ASSERT_TRUE(status.IsOK()) << "error msg: " + status.msg;
+    }
+    HandleSQL("create database test1;");
+    HandleSQL("use test1;");
+    std::string create_sql = "create table t1(c1 string, c2 int, c3 double);";
+    sr->ExecuteSQL(create_sql, &status);
+    ASSERT_TRUE(status.IsOK()) << status.msg;
+    HandleSQL("insert into t1 values ('aab', 11, 1.2);");
+    std::string so_path = openmldb::test::GetParentDir(openmldb::test::GetExeDir()) + "/libtest_udf.so";
```

Comment:
should ensure `libtest_udf.so` was built before `sql_cmd_test`

Reply:
`libtest_udf.so` has been built before `sql_cmd_test` run.